### PR TITLE
Added Doc Strings for the functions in "helpers.py" and "infer.py"

### PR DESCRIPTION
**Pull request by Sumanth077**
Hi @paxcema, Updated the doc strings for most of the functions. Kindly let me know the changes to be made, if any.

### CODE REVIEW

File: type_infer/helpers.py [hunk 1]
```diff
@@ -247,6 +290,9 @@ def decontracted(phrase):
 
 
 def contains_alnum(text):
+    """
+    The function takes in a string and checks if any of its characters are alphanumeric, returning True if found and False if not.
```
Comment (paxcema) [inline type_infer/helpers.py]: Rephrase as:

> Check and return whether any of the input string characters are alphanumeric.

**Review by paxcema — CHANGES_REQUESTED** [review]
Looks really good! I have left one minor request, but once you've addressed it we can merge this :)

File: type_infer/helpers.py (hunk 1 above)
Comment (Sumanth077) [reply type_infer/helpers.py]: Updated it @paxcema.

**Review by paxcema — APPROVED** [review]

**Review by paxcema — CHANGES_REQUESTED** [review]
Lint checks are failing. Could you please add `  # noqa` to the end of all new docstrings?

**Comment by Sumanth077** [comment]
Updated it @paxcema. Could you kindly check now.

**Comment by paxcema** [comment]
@Sumanth077 apologies! I somehow forgot to comment on this, but it looks OK! 

Could you perhaps merge `staging` against it to solve conflicts, so that we can merge?

**Comment by paxcema** [comment]
Closing for now. Feel free to reopen whenever you have some time @Sumanth077!

**Pull request closed without merging**